packages/core/contracts/financial-templates/funding-rate-store/implementation/FundingRateStore.sol

```diff
@@ -302,6 +479,34 @@ contract FundingRateStore is FundingRateStoreInterface, Testable, Lockable {
         store.payOracleFeesErc20(address(collateralCurrency), amount);
     }
 
+    function _calculateProposalRewardPct(
+        address perpetual,
+        uint256 startTime,
+        uint256 endTime,
+        FixedPoint.Signed memory proposedRate,
+        FixedPoint.Signed memory currentRate
+    ) private view returns (FixedPoint.Unsigned memory reward) {
+        uint256 timeDiff = endTime.sub(startTime);
+
+        FixedPoint.Unsigned memory rewardRate = _getFundingRateRecord(perpetual).rewardRatePerSecond;
+
+        // First compute the reward for the time elapsed.
+        reward = rewardRate.mul(timeDiff);
+
+        // Next scale the reward based on the absolute difference % between the current and proposed rates.
```

Comment:
Can you write a one-line equation that shows the computation being done below in plain math?

Something like

```
// reward = (proposedRate - currentRate) / currentRate ....
```

Reply:
`reward = reward * (1 + (proposed - current) / current)`

Reply:
Or if `current = 0` then `reward = reward * (1 + (proposed - current))`

Reply:
I will add this to the comments

Reply:
FYI we need to change the `withdrawFundingRateRewards` interface to return the actual amount of rewards withdrawn, in case the regular fees reduce the `PfC` such that the caller cannot withdraw their full amount.